project-canvas/src/components/StoryMapView/AddLevel.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from "@mantine/core"
+import { Updater } from "use-immer"
+import { getAllActions, getRndInteger } from "./helpers/utils"
+import { Case, SubActionLevel } from "./types"
+
+export function AddLevel({
+  setLevels,
+  setCases,
+}: {
+  setLevels: Updater<SubActionLevel[]>
+  setCases: Updater<Case[]>
+}) {
+  return (
+    <Button
+      onClick={() => {
+        const levelId = `level-${getRndInteger()}`
```

Comment:
probably i am mistaken and i know the chances are eventually very low, but could this not lead to duplicated IDs?

Reply:
It definitely could. That is a placeholder for now, until the structure is finalized and then will implement a proper ID system